react/demo/src/app/samples/toast/Action.route.tsx

```diff
@@ -0,0 +1,69 @@
+import type {ToastApi, ToastContext, ToastProps} from '@agnos-ui/react/components/toast';
+import {Toast} from '@agnos-ui/react/components/toast';
+import {WidgetsDefaultConfig} from '@agnos-ui/react/config';
+import {Slot} from '@agnos-ui/react/slot';
+import type {AdaptPropsSlots} from '@agnos-ui/react/types';
+import biArrowCounterClockwise from 'bootstrap-icons/icons/arrow-counterclockwise.svg?raw';
+import biCheckCircleFill from 'bootstrap-icons/icons/check-circle-fill.svg?raw';
+import {useRef, useState} from 'react';
+
+const ActionToastDemo = () => {
+	const [toastConfig] = useState<Partial<AdaptPropsSlots<ToastProps>>>({
+		dismissible: true,
+		className: 'text-bg-success',
+		autohide: false,
+		slotStructure: ToastAction,
+	});
+	const refToast = useRef<ToastApi>(null);
+
+	function reset() {
+		refToast.current?.open();
+	}
+
+	return (
+		<>
+			<p> To put the action in the toast simply create custom contents.</p>
+			<p>
+				<strong>Note:</strong> When header is not present default display is <code>flex</code>
+			</p>
+			<div className="col-6">
+				<WidgetsDefaultConfig toast={toastConfig}>
+					<Toast ref={refToast}>This is a toast with action</Toast>
+				</WidgetsDefaultConfig>
+			</div>
+			<button className="btn btn-primary mt-2" onClick={reset}>
+				Reset
+			</button>
+		</>
+	);
+};
+
+export default ActionToastDemo;
+
+const ToastAction = ({widget, state}: ToastContext) => {
+	function actionDemo() {
+		window.alert('Undo');
+	}
+
+	return (
+		<>
+			<div className="d-flex w-100">
+				<div className="d-flex align-items-center flex-grow-1 toast-body">
+					<span className="d-flex me-2" dangerouslySetInnerHTML={{__html: biCheckCircleFill}}></span>
```

Comment:
Once https://github.com/AmadeusITGroup/AgnosUI/pull/478 is merged, remove the usage of ```dangerouslySetInnerHTML```